<ui version="4.0" >
 <class>Vocabulary</class>
 <widget class="QWidget" name="WordList" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Vocabulary</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <property name="spacing" >
        <number>4</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="lbVocab" >
           <property name="text" >
            <string>Vocabulary:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KLineEdit" name="leSearch" >
           <property name="clickMessage" >
            <string>Filter</string>
           </property>
           <property name="showClearButton" stdset="0" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="DragTableView" name="tvVocab" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>255</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" native="1" name="widget" >
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QCheckBox" name="cbShowCompleteLexicon" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="text" >
             <string>Include unused Words from the Shadow Lexicon</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" native="1" name="wgSettings" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="KPushButton" name="pbImport" >
            <property name="text" >
             <string>Import Dictionary</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="KPushButton" name="pbRemoveWord" >
            <property name="text" >
             <string>Remove Selected Word</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>4</number>
       </property>
       <property name="leftMargin" >
        <number>3</number>
       </property>
       <property name="bottomMargin" >
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text" >
          <string>Train Words:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DropListWidget" name="lwTrainingWords" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="pbAddToTraining" >
         <property name="text" >
          <string>Add to Training</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="pbDeleteTrainingWord" >
         <property name="text" >
          <string>Remove Word from Training</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="pbTrainList" >
         <property name="toolTip" >
          <string>Starts the Training of the selected Words</string>
         </property>
         <property name="text" >
          <string>Train selected Words</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>DropListWidget</class>
   <extends>KListWidget</extends>
   <header>droplistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>DragTableView</class>
   <extends>QTableView</extends>
   <header>dragtableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
